src/plugins/intel_cpu/src/nodes/input.cpp

```diff
@@ -262,6 +393,90 @@ void Input::cloneBlobIfRequired() {
         needFlushDenormalsToZero = false;
     }
 
+    // The presence of subnormals is better to determined at IR read time.
+    auto checkSubnormalsAndBF16Overflows = [&](bool& has_subnormals, bool& has_bf16_overflows) {
+        if (prec == ov::element::f32) {
+            uint32_t const* u32data = m_constOp->get_data_ptr<uint32_t>();
+            float const* f32data = m_constOp->get_data_ptr<float>();
+
+            if (!size)
+                return;
+
+            // Only LLMs scalar constant nodes with bf16 inferencePrecision need to be checked for saturation
+            const bool do_bf16_saturation_check =
+                (context->getConfig().inferencePrecision == ov::element::bf16 && size == 1) ? true : false;
```

Comment:
Why? The fact we have seen overflows only for scalar constants before, doesn't mean it might be in vectors or matrices. I would say we need to check all constants to guarantee BF16 will be valid.

Reply:
size == 1(scalar) limitation removed